00-programming-fundamentals/03-installfest.md

```diff
@@ -11,7 +11,9 @@ Xcode is an **integrated development environment** (often called an **IDE**) for
 **You should have installed Xcode Command-Line tools in JumpStart.**
 
 ### Trust but Verify
-Run `$ xcode-select -p`. If it tells you `/Library/Developer/CommandLineTools`, you're ready to move on.
+
+If the Xcode Command-Line tools are not installed you can install it with:
+`xcode-select -p` and be patient because it will take time to install.
```

Comment:
Apologies, it looks like I was incorrect with my earlier comment.

Students need to do the following:
1. Run `xcode-select -p` to display the current path of the installed Xcode command line tools.
1. IF the previous command displayed a directory path, then they can move on to the Homebrew instructions.
1. IF the previous command displayed something like: `xcode-select: error: unable to get active developer directory, use xcode-select --switch to set one (or see man xcode-select)` then they should run the command `xcode-select --install` before moving on.